packages/documentation/docs/integration/flow-of-funds.md

```diff
@@ -0,0 +1,119 @@
+# Flow of funds
+
+Rafiki communicates with the Account Servicing Entity over webhook events, which are defined in the _Getting Started_ section. Each event requires the Account Servicing Entity to either deposit or withdraw liquidity into or from Rafiki. This document will provide an example flow for each of the defines webhook events.
+
+## `incoming_payment.created`
+
+The `incoming_payment.created` event indicates that an incoming payment has been created. It has not yet received any funds so no action is required.
+
+```mermaid
+sequenceDiagram
+
+participant ASE as Account Servicing Entity
+participant R as Rafiki
+
+R->>ASE: webhook event: incoming payment created
+ASE->>ASE: no action required
+```
+
+## `incoming_payment.completed`
+
+The `incoming_payment.completed` event indicates that an incoming payment has been completed, either automatically or manually, and that any funds that have been received into this incoming payment should be withdrawn and credited to the recipient's account with the Account Servicing Entity.
+
+Example: An incoming payment was completed and received $10.
+
+```mermaid
+sequenceDiagram
+
+participant ASE as Account Servicing Entity
+participant R as Rafiki
+
+R->>ASE: webhook event: incoming payment completed,<br>receiveAmount: $10
+ASE->>R: admin API call: WithdrawEventLiquidity
+ASE->>ASE: credit receiver's account with $10
+```
+
+## `incoming_payment.expired`
+
+The `incoming_payment.expired` event indicates that an incoming payment has expired, and that any funds that have been received into this incoming payment should be withdrawn and credited to the recipient's account with the Account Servicing Entity.
+
+Example: An incoming payment has expired and received $2.55.
+
+```mermaid
+sequenceDiagram
+
+participant ASE as Account Servicing Entity
+participant R as Rafiki
+
+R->>ASE: webhook event: incoming payment expired,<br>receiveAmount: $2.55
+ASE->>R: admin API call: WithdrawEventLiquidity
+ASE->>ASE: credit receiver's account with $2.55
+```
+
+## `outgoing_payment.created`
+
+The `outgoing_payment.created` event indicates that an outgoing payment has been created and is awaiting liquidity. The Account Servicing Entity should put a hold on the sender's account and deposit the funds into Rafiki.
+
+Example: An outgoing payment for $12 has been created.
+
+```mermaid
+sequenceDiagram
+
+participant ASE as Account Servicing Entity
+participant R as Rafiki
```

Comment:
does it make sense to have this as "sending Rafiki"?

Reply:
Should we distinguish between sending and receiving? It may be confusing and people could think that there are two types of Rafiki

Reply:
I don't think labeling this as "sending Rafiki" is necessary  since it's simply a part of the integration process. Currently, the ASE will only receive webhook events from one Rafiki instance, which is the one they intend to integrate.